src/main/java/seedu/sellsavvy/model/order/OrderList.java

```diff
@@ -27,19 +28,32 @@ public boolean contains(Order toCheck) {
         return internalList.stream().anyMatch(toCheck::isSameOrder);
     }
 
-    //TODO:
     /**
      * Replaces the first instance of an equivalent order {@code target} in the list with {@code editedOrder}.
      * {@code target} must exist in the list.
      */
-    public void setOrder(Order target, Order editedOrder) {}
+    public void setOrder(Order target, Order editedOrder) {
+        requireAllNonNull(target, editedOrder);
+
+        int index = internalList.indexOf(target);
+        if (index == -1) {
+            throw new OrderNotFoundException();
+        }
+
+        internalList.set(index, editedOrder);
+    }
 
-    //TODO:
     /**
      * Replaces the contents of this list with {@code orders}.
      */
-    public void setOrders(List<Order> orders) {}
-    public void setOrders(OrderList replacement) {}
+    public void setOrders(List<Order> orders) {
+        requireAllNonNull(orders);
+        internalList.setAll(orders);
+    }
+    public void setOrders(OrderList replacement) {
```

Comment:
is this missing a javadoc and space?

Reply:
You mean whitespace? Javadoc not provided similar to UniquePersonList.setOrders, assumed that command is self-explanatory.